doc/storage/api/api.rst

```diff
@@ -658,15 +842,193 @@ These definitions must be defined in the header file :file:`psa/protected_storag
 
     * ``capacity`` unchanged.
 
+    * Data in the ranges 0 to ``data_offset`` is not modified.
 
+    * If ``data_offset + data_length < size`` then data in the range ``data_offset + data_length` to `size`` is not modified.
 
-.. function:: psa_ps_get_support
 
-    .. return:: uint32_t
+
+.. function:: psa_ps_rename
```

Comment:
Is there a reason why rename is not an API that we want to permit on the ITS storage?